### PR TITLE
Add message source display + fix empty text with plain text emails with attachment

### PR DESCRIPTION
**Pull request by repli2dev**
Sometime it is useful to see complete message including all headers. This pull request adds this functionality via "Source" link which opens a new blank window with message source.

### CODE REVIEW

File: src/MailPanel.latte [hunk 1]
```diff
@@ -87,12 +88,20 @@
...
+							{var $mailContent = '<pre style="font-family:serif">' . \Nextras\MailPanel\MailPanel::extractPlainText($mail) . '</pre>'}
+						{else}
+							{var $mailContent = '<pre style="font-family:serif">' . $mail->getBody() . '</pre>'}
+						{/if}
+
+						Plaintext e-mail
+					{else}
+						HTML e-mail
+					{/if}
+					<a class="source-link" target="_blank" href="{$baseUrl}?mail-panel=source&mail-panel-mail={$index}">Source</a>
```
Comment (hrach) [inline src/MailPanel.latte]: &amp;

File: src/MailPanel.latte [hunk 2]
```diff
@@ -87,12 +88,20 @@
 				{/if}
 				{* Little magic here. Create iframe and then render message into it (needed because HTML messages) *}
 				{var $mailContent = $mail->getHtmlBody()}
-				{if $mailContent == NULL}
-					{var $mailContent = '<pre style="font-family:serif">' . $mail->getBody() . '</pre>'}
-					<tr><th colspan="2">Plaintext e-mail</th></tr>
-				{else}
-					<tr><th colspan="2">HTML e-mail</th></tr>
-				{/if}
+				<tr><th colspan="2">
```
Comment (hrach) [inline src/MailPanel.latte]: better indent?

**Comment by hrach** [comment]
Overall, I'm ok with it, just fix the indent & amp please :)

**Comment by repli2dev** [comment]
Ok, thanks for comments, should be ok now.

**Comment by repli2dev** [comment]
Ok, missed the fact that there were multiple & entities to fix.
Moreover I have added similar functionality to open message in new window (very good for mail template development and tuning).

**Comment by hrach** [comment]
thanks

**Pull request merged**